.env

```diff
@@ -1,3 +1,7 @@
+ARCHIVE_BUCKET=fake-archive-bucket
+INCOMING_BUCKET=fake-bucket
+AWS_ACCESS_KEY_ID=
```

Comment:
I wonder whether in the AWS credentials case it would be better to mount `~/.aws/credentials`? (the `boto` library also supports that)

Reply:
Yes that should work. I will test it out and include it if all goes well in the next pr 👍